docs/bokeh/source/index.rst

```diff
@@ -88,8 +88,9 @@ You can also find more information about Bokeh on `Twitter`_, `Medium`_, and
 .. image:: _images/bokeh-hero.png
     :width: 100%
     :alt: A collage of 36 thumbnails of Bokeh plots
+    :target: _galery
 
-
+.. _galery: https://docs.bokeh.org/en/latest/docs/gallery.html
```

Comment:
This link should also be a relative link, so that if someone is viewing the docs for a specific version, they get directed to the gallery for that version, and not to "latest"

Reply:
got it thanks!

But I I'm not sure how to do that, probably there is a keyword or a variable in rst that I'm not finding, I have literally zero experience with it.
 
I just found *BOKEH_CURRENT_VERSION* as a variable in conf.py, can we use that here? I couldn't make it work. Maybe somebody can tell me where to look, thanks in advance.

Reply:
I actually just realized my suggestion is not a good one. A relative link would still have the `en/<version>` in it. I think the PR as is is probably the best we can do without a lot more work.